gatsby-config.js

```diff
@@ -204,6 +204,13 @@ module.exports = {
         path: `./content/md/`,
       },
     },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `homepage-content`,
+        path: `./content/homepage/`,
+      },
+    },
```

Comment:
it would be great to respect the project structure, the new homepage content is now out of `md` folder, could we move that there md/en/homepage and remove the additional not needed config